FAR-Fullstack/PrimeTime
# Creating controllers views

What I did:
- CRUD for past_event + views
- CRUD for future_event + views

for now, it is a new page for all actions. We will have to implement the index of past and future events on the home page, by sections...

How to test:
- db:migrate db:seed
- login from the footer, "Admin corner" with email: "god@godmode.com", password: "password"
- see that now you have links from navbar:
- try to create a new future_event, edit it and delete it
- try to create a new past_event, edit it and delete it
-see that you have links in the #events section
